libqtile/layout/zoomy.py

```diff
@@ -138,11 +138,14 @@ def info(self):
         return d
 
     def focus(self, win):
-        if self.focused and self.property_name:
+        if self.focused and self.property_name and self.focused.window.get_property(
```

Comment:
Can we make this `is None`, that way people can set it to "" if they want? Other than that, this patch looks good to me.